packages/core/src/Stores/client-store.js

```diff
@@ -2729,6 +2731,18 @@ export default class ClientStore extends BaseStore {
         }
     }
 
+    passkeysTrackActionEvent({ action, subform_name, error_message }, is_effortless_modal = false) {
+        const event_name = is_effortless_modal ? 'ce_passkey_effortless_form' : 'ce_passkey_account_settings_form';
+
```

Comment:
Instead of having a common function and conditionals inside., IMO its more readable in their own components.